### ISSUE TITLE
Add support for Pine script v6

### ISSUE THREAD

**Issue by claudio-silva** [frizLabz-FFriZz/Pine-Script-v5-VS-Code#12]
Congratulations for this amazing extension!
This is definitely the VSCode extension that TradingView should have done!
Now I can really work offline as if I'm working on the TradingView editor, with all the expected features.
This allows me to use Git, AI assistance, etc, which are not available on the platform, which has a lot of advantages.
Outstanding!
But now that TradingView has released a new language version, it is starting to lag behind the current language capabilities.
There were no major changes to the language syntax in this version, so it should not be too difficult to make it compatible and not display an error on the `//@version=6` line.
Although, if you do add that support, you should probably also change the name of the extension from `Pine-Script-v5-VS-Code` to `Pine-Script-v6-VS-Code`, or even better, `Pine-Script-VS-Code` to make it future proof and language version independent.

**Comment by kaigouthro** [comment]
Agree. :) noted. I've just joined on but after i give a proper gander through all the wrings and catch my bearings, i will be doing such things. =)

**Comment by stanislavlysenko0912** [comment]
Hello @kaigouthro, will there be an update?
I've explored the repository quite a bit, I take it the main problem is parsing the updated documentation for the `pineDocs.json` file?

Or something else?

**Comment by kaigouthro** [comment]
> Hello [@kaigouthro](https://github.com/kaigouthro), will there be an update? I've explored the repository quite a bit, I take it the main problem is parsing the updated documentation for the `pineDocs.json` file?
> 
> Or something else?

well, for now, i've updated and it works, i'll tackle the docs part of it later, but it's fairly on par as is with 5/6.. should be fine.. new theme i put in there, and markdown code block highlighting now in md files =)
